### PR TITLE
the input of Funcs here is useless, i think it should be deleted.

### PR DESCRIPTION
**Pull request by liangwenguo**
**What this PR does / why we need it**:
the input of Funcs is useless, i think it should be deleted. I find it in other function, then no parameter sended to Funcs.
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 )
 
 // Funcs returns the fuzzer functions for the kubeadm apis.
-func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
+func Funcs() []interface{} {
```
Comment (resouer) [inline cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer.go]: cc @luxas take a look? It's a minor fix.

**Comment by liangwenguo** [comment]
@timothysc PTAL, Thank you

**Comment by liangwenguo** [comment]
keep-open

**Review by luxas — APPROVED** [review]
/lgtm

but bazel is failing, please fix

**Comment by liangwenguo** [comment]
i found that the return of Funcs() can not be used as the parameter of MergeFuzzerFuncs().

File: cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer.go [hunk 2]
```diff
@@ -17,110 +17,48 @@ limitations under the License.
...
-				MountPath: "foo",
-			}}
-			obj.Etcd.ExtraArgs = map[string]string{"foo": "foo"}
-			obj.Etcd.SelfHosted = &kubeadm.SelfHostedEtcd{
-				CertificatesDir:    "/etc/kubernetes/pki/etcd",
-				ClusterServiceName: "etcd-cluster",
-				EtcdVersion:        "v0.1.0",
-				OperatorVersion:    "v0.1.0",
-			}
-			obj.KubeletConfiguration = kubeadm.KubeletConfiguration{
```
Comment (luxas) [inline cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer.go]: Why remove all this fuzzing instead of just changing the signature?

Comment (liangwenguo) [reply cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer.go]: there is something wrong when i copy the code, i will fix it.

**Comment by errordeveloper** [comment]
@liangwenguo are you planning on getting back to this?

**Comment by resouer** [comment]
kindly ping @liangwenguo for another time :)

**Comment by liangwenguo** [comment]
@errordeveloper  so sorry, can not give you a reply in time, i am working on it

**Comment by liangwenguo** [comment]
@timothysc @resouer @luxas @fejta-bot so sorry, i found that the code structure has been changed, my work has made a small optimization, but it may make the code unstable, so i give up wy work.

**Comment by errordeveloper** [comment]
@liangwenguo thanks!

**Pull request closed without merging**